PySight.py

```diff
@@ -617,7 +598,7 @@ def misp_check_for_previous_events(misp_instance, isight_alert):
             try:
                 event = misp_instance.new_event(0, 2, 0, isight_alert.title + " pySightSight " + isight_alert.reportId,
```

Comment:
A bit confused here. We are creating an event on the server here vs. doing the in-memory construction of the `MISPEvent` in `is_map_alert_to_event` (https://github.com/deralexxx/PySight2MISP/pull/6/files#diff-064503d379390dc031f013232f9d9354R349)

Reply:
new_event were mistakes